### ISSUE TITLE
Labels overlap for an archived product with zero quantity

### ISSUE THREAD

**Issue by efalayi** [reactioncommerce/reaction#2826]
### Expected behavior
Product label should clearly show that the product has zero quantity and has been archived with no overlap

### Actual Behavior
Labels for 'archived' status and 'sold out' overlap

### Steps to Reproduce the Behavior
Create a product and fill necessary fields
Specify quantity as zero
Archive the prooduct

### Versions
```
Node: 8.3.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.10.0
Reaction: 1.5.0
Reaction branch: marketplace
```

<img width="1213" alt="screen shot 2017-09-11 at 12 54 05 pm" src="https://user-images.githubusercontent.com/28438068/30278068-644fcf28-9701-11e7-86ca-dc9e8d3a55bd.png">

**Comment by rymorgan** [comment]
related to #2801

**Comment by kieckhafer** [comment]
As mentioned, this is a duplicate of #2801 , and will be closed in the pending PR #2887 .
